perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -1555,14 +1565,71 @@ std::vector<PredictedRefPath> MapBasedPredictionNode::getPredictedReferencePath(
     object.kinematics.twist_with_covariance.twist.linear.x,
     object.kinematics.twist_with_covariance.twist.linear.y);
 
+  // Using a decaying acceleration model
+  // a(t) = obj_acc - obj_acc(1-e^(-λt)) = obj_acc(e^(-λt))
+  const double obj_acc = (use_vehicle_acceleration_)
+                           ? std::hypot(
+                               object.kinematics.acceleration_with_covariance.accel.linear.x,
+                               object.kinematics.acceleration_with_covariance.accel.linear.y)
+                           : 0.0;
+  // The decay constant λ = ln(2) / exponential_half_life
+  const double T = prediction_time_horizon_;
+  const double exponential_half_life = acceleration_exponential_half_life_;
```

Comment:
This line can be removed since `exponential_half_life` is only used once in the next line.